simulation/src/types.rs

```diff
@@ -0,0 +1,107 @@
+use std::{cmp::Ordering, collections::VecDeque, fmt::Debug};
+
+use deckofcards::{Card as DOCCard, Rank, Suit};
+use uuid::Uuid;
+
+#[derive(Debug)]
+pub struct GameState {
+    pub table: VecDeque<Player>,
+    pub top_card: Option<Vec<Card>>,
```

Comment:
Add new type for "unit played from hand" (e.g. enum allowing for single, pair, triple, or quad)